RELEASE.md

```diff
@@ -0,0 +1,34 @@
+# Release
+
+## On a fork
+
+Please follow these instructions carefully.
+Ensure that you name the branch precisely as `release-vX.Y.Z`
+since this nomenclature is crucial for obtaining the `future_version` in the changelog.
+Your attention to this specific branch naming convention is essential for accurate version tracking in the changelog.
+
+```shell
+export RELEASE_VERSION="X.Y.Z"
+git switch main
+git pull -r
+git switch -c release-v$RELEASE_VERSION
+
+bundle config set --local path vendor/bundle
+bundle config set --local with 'release'
+bundle install
+
+CHANGELOG_GITHUB_TOKEN="token_MC_tokenface" bundle exec rake changelog
+git commit -am "Release v${RELEASE_VERSION}"
+git push origin release-v$RELEASE_VERSION
```

Comment:
I'm always a fan of setting up tracking and reuse the branch you already have locally
```suggestion
git push --set-upstream origin HEAD
```